docs/src/AlgebraicGeometry/ToricVarieties/CohomologyClasses.md

```diff
@@ -12,20 +12,23 @@ DocTestSetup = Oscar.doctestsetup()
 
 ### General constructors
 
-Cohomology classes are elements of the cohomology ring, which is a quotient ring (for normal toric varieties that
-are simplicial and complete). All of the following methods accept an optional argument `quick` (default: `false`).  
-When set to `true`, computations are performed in the base ring of the cohomology ring to improve performance.  
-This is especially useful when constructing cohomology classes on large toric varieties, where operations
-in the quotient ring - such as Gröbner basis computations - can be computationally expensive.
+Cohomology classes are elements of the cohomology ring, which is a quotient ring for normal toric varieties that
+are simplicial and complete. All of the following methods accept an optional argument `completeness_check`
+(default: `true`).  
 
-**Caveat:** When `quick = true`, the method will not detect whether the resulting cohomology class is trivial.
-Use this option when faster computations are preferred and triviality checks are not required.
+By default, completeness is checked to ensure mathematical correctness. However, these checks can be computationally
+expensive—especially for large toric varieties, where operations in the quotient ring (such as Gröbner basis
+computations) are already costly. To speed up computations, you may disable this step by passing
+`completeness_check = false`.
+
+**Caveat:** Disabling the completeness check skips verification that the underlying toric variety is complete.
+Use this option only if completeness is already known or guaranteed, and faster computations are preferred.
 
 ```@docs
-cohomology_class(v::NormalToricVarietyType, p::MPolyQuoRingElem)
-cohomology_class(d::ToricDivisor)
-cohomology_class(c::ToricDivisorClass)
-cohomology_class(l::ToricLineBundle)
+cohomology_class(v::NormalToricVarietyType, p::MPolyQuoRingElem; completeness_check::Bool = true)
+cohomology_class(d::ToricDivisor; completeness_check::Bool = true)
+cohomology_class(c::ToricDivisorClass; completeness_check::Bool = true)
+cohomology_class(l::ToricLineBundle; completeness_check::Bool = true)
```

Comment:
you can drop all the kwargs from these `@docs` lists (also below in lines 72-84), as kwargs are not used for method dispatch